components/data-collection/woopra.mdx

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
weird, I had merge this new page yesterday: https://github.com/edgee-cloud/docs/pull/72

Reply:
right. I keep yours